app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventCapture/EventCapturePresenterImpl.kt

```diff
@@ -109,6 +111,7 @@ class EventCapturePresenterImpl(
         errorFields: List<FieldWithIssue>,
         emptyMandatoryFields: Map<String, String>,
         warningFields: List<FieldWithIssue>,
+        eventMode: EventMode?,
```

Comment:
is this `attemptFinish` method covered by test all the possible outcomes?, maybe we should do it in order to validate all the expected outputs